src/main/webapp/search.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset = "UTF-8">
```

Comment:
Remove the spaces surrounding the equal sign. Everything looks great!

Reply:
Lol ironic, thanks!